src/Microsoft.Sbom.Contracts/Contracts/SBOMValidationResult.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
```

Comment:
I created a new file instead of reusing the SBOMGenerationResult is because:

I would break existing customer of the generator API by doing that.
In the future validation and generation APIs could return different objects, and I feel like it's logical to separate out the two objects.